plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/H2QueryRunner.java

```diff
@@ -58,6 +58,7 @@ public static DistributedQueryRunner createH2QueryRunner(Iterable<TpchTable<?>>
     public static DistributedQueryRunner createH2QueryRunner(Iterable<TpchTable<?>> tables, Map<String, String> properties, Module module)
             throws Exception
     {
+        DistributedQueryRunner.setupLogging();
```

Comment:
Can we make this part of `DistributedQueryRunner.Builder`, so your config would be default while user could reset it to plain `Logging.initialize()` or maybe configure in a custom way?

Reply:
The reason I did it outside is that in some test classes logging is set up once in a `@BeforeClass` method and there are multiple query runners created in tests. The loggers for which this configures levels are not tied to a particular query runner, so it might be surprising they keep changing after creating new query runners.

Reply:
Moved to the constructor.